templates/master/00-master/gcp/units/openshift-gcp-routes.service.yaml

```diff
@@ -7,6 +7,7 @@ contents: |
   ConditionKernelCommandLine=|ignition.platform.id=gcp
   Wants=network-online.target
   After=network-online.target
+  PartOf=network.target
```

Comment:
I'm far of a systemd expert, but should this After=network.target?
> Services using the network should hence simply place an After=network.target dependency in their unit files, and avoid any Wants=network.target or even Requires=network.target.
https://www.freedesktop.org/wiki/Software/systemd/NetworkTarget/